data.sql

```diff
@@ -4,3 +4,12 @@ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Gabumon', '2018-11-15', 2, true, 8);
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Pikachu', '2021-01-07', 1, false, 15.04);
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Devimon', '2017-05-12', 5, true, 11);
+
+
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Charmander', '2020-02-08', 0, false, -11);
```

Comment:
- Hey, the datatype for the weight is a decimal, let's kindly make the value a floating number `-11.0`